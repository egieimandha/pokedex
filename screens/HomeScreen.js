import React from "react"
import { StyleSheet } from "react-native"

import { RootComponent, Section, Item } from "../components/index"
import BodyBold from "../components/UI/BodyBold"

const HomeScreen = props => {
  return (
    <RootComponent>
      <Section>
        <Item center>
          <BodyBold>Lalapopo</BodyBold>
        </Item>
      </Section>
    </RootComponent>
  )
}

const styles = StyleSheet.create({
  screen: {
    flex: 1
  }
})

export default HomeScreen
