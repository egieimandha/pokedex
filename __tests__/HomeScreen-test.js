import React from "react"
import "react-native"
import HomeScreen from '../screens/HomeScreen'
import renderer from 'react-test-renderer'

test('Home Screen snapshot', () => {
  const snap = renderer.create(
    <HomeScreen />
  ).toJSON()
  expect(snap).toMatchSnapshot()
})