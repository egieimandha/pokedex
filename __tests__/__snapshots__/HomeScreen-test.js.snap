// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Screen snapshot 1`] = `
<RNCSafeAreaView
  onInsetsChange={[Function]}
  style={
    Object {
      "flex": 1,
    }
  }
/>
`;
